docs/Classes.html

```diff
@@ -13,7 +13,7 @@
     <a title="Classes  Reference"></a>
```

Comment:
From here on there is only generated documentation, which just mirrors the inline documentation. Just ignore it.